news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.8.2"
+subtitle: "Maintenance version of the 1.7.8.x branch"
+date:   2021-12-07 14:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2021/12/1.7.8.2_banner.jpg
+icon: icon-checked
+tags:
+ - version
+ - patch
+ - releases
+ - 1.7.8.x
+ - 1.7
+
+---
+
+The ongoing work on security for PrestaShop software continues. We have identified and fixed a new security issue, it has been decided to deliver a new maintenance release for 1.7.8.X branch.
+
+*![1.7.8.2 is available!](/assets/images/2021/12/1.7.8.2_banner.jpg)*
+
+This release also fixes 1 regression reported on version 1.7.8.X.
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before.
+
+Reminder: the 1-Click Upgrade module’s latest version is v4.13.0, don’t forget to upgrade it.
+
+
+## Main fix
+
+Below is the one regression issue we fixed in this version :
+
+- Fix wishlist button disappearance on the category page while using any filter in faceted search ([#26327](https://github.com/PrestaShop/PrestaShop/issues/26327))
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.2).
```

Comment:
I think we usually use the changelog uploaded on the .com, but maybe you're right to use the one on github now that we try to separate things.
```suggestion
Read the [full changelog here](https://download.prestashop.com/download/releases/changelog_1.7.8.2.txt).
```